### DISCUSSION TITLE
[Desktop] Windows -  replace native title bar with custom dark styled one

### DISCUSSION THREAD

**Discussion by iamtheoneaboveall** [wireapp/wire-desktop#5072] [Ideas]
Replace the ugly looking default native white colored titlebar with a colored dark style like Discord app has.

![dark titlebar](https://i.imgur.com/kmsy2fJ.png)

**Comment by nynhex** [comment]
@iamtheoneaboveall please submit a pr on this if you have interest.  I feel it will help the team.

Also because not everyone likes discord style themes, this should be something that the community agrees on or perhaps something that can be themed and swapped on demand.
